src/explorer/weights/WeightSlider.js

```diff
@@ -33,11 +38,19 @@ export class WeightSlider extends React.Component<Props> {
   }
 }
 
+function convertWeightToSliderValue(weight: number): number {
+  return weight === 0 ? WEIGHT_SLIDER_MIN : Math.log2(weight);
+}
+
+function convertSliderValueToWeight(sliderValue: number): number {
+  return sliderValue === WEIGHT_SLIDER_MIN ? 0 : 2 ** sliderValue;
```

Comment:
Could you please add docs to this function to clarify its contract?

  - Are clients allowed to call it with values outside the range set by
    `WEIGHT_SLIDER_MIN` and `WEIGHT_SLIDER_MAX`?
  - If this is disallowed, what should happen if a client does this?
    Should you throw an error? Return `NaN`? Is the behavior undefined?

Currently, the function is discontinuous at `WEIGHT_SLIDER_MIN`, which
is a bit strange. Depending on what you want the semantics to be, you
might s/===/<=/ or you might add some error checks.

Reply:
Added docs, and also added error handling.